tests/testapp/home/static/button_entity.js

```diff
@@ -0,0 +1,6 @@
+const ButtonDecorator = ({ entityKey, children }) => {
+  const attrs = {'data-tooltip': entityKey, className: 'RichEditor-button'};
+  return window.wagtaildraftail.createElement('span', attrs, children);
+};
```

Comment:
I thought we wanted to allow an easy path, but maybe that's too easy.

To remove this though, I'll need help with the webpack config, so it has 2 entry points (the admin and the test sites) and therefore 2 exit points.